lib/jekyll-github-metadata/ghp_metadata_generator.rb

```diff
@@ -34,19 +34,39 @@ def git_remote_url
         end.uniq.first || ""
       end
 
-      def git_nwo
+      def nwo_from_git_origin_remote
         return unless Jekyll.env == "development"
         matches = git_remote_url.match %r{github.com(:|/)([\w-]+)/([\w-]+)}
```

Comment:
To note, this will only work with .com, not Enterprise. Maybe we should ignore the host?